src/Service/Operation/ListEpreuveOperation.php

```diff
@@ -0,0 +1,46 @@
+<?php declare(strict_types=1);
+
+namespace Alamirault\FFTTApi\Service\Operation;
+
+use Alamirault\FFTTApi\Model\Epreuve;
+use Alamirault\FFTTApi\Service\FFTTClientInterface;
+
+final class ListEpreuveOperation
+{
+    public function __construct(
+        private readonly FFTTClientInterface $client,
+    ) {}
+
+    /**
+     * @param string $type (E : épreuves par équipe, I: épreuves individuelles)
```

Comment:
I think for now services we can do better (and migrate old approach ?):

Create two const `public const TYPE_EQUIPE = 'E' ` and `public const TYPE_INDIVIDUEL = 'I' `

Update phpdoc, `@param self::TYPE_* $type` https://phpstan.org/writing-php-code/phpdoc-types#literals-and-constants

Then in function, throw exception is not in const array. (No more implicit behavior)

Reply:
We can also use enum https://stitcher.io/blog/php-enums, `EpreuveEnum`. If we choose this we can remove in_array check + type `Epreuve->typeEpreuve` + use this type in `FFTTAPI->listEpreuves()` arg